RecipeMatch/View Controllers/ProfileViewController.m

```diff
@@ -9,18 +9,96 @@
 #import "Parse/Parse.h"
 #import "LoginViewController.h"
 #import "SceneDelegate.h"
+#import "GridRecipeCell.h"
+#import "LikedRecipe.h"
+#import "UIImageView+AFNetworking.h"
 
-@interface ProfileViewController ()
-
+@interface ProfileViewController () <UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout>
+@property (weak, nonatomic) IBOutlet UICollectionView *recipesCollectionView;
+@property (nonatomic, strong) NSArray *recipes;
+@property (weak, nonatomic) IBOutlet UICollectionViewFlowLayout *flowLayout;
+@property (nonatomic, strong) UIRefreshControl *refreshControl;
 @end
 
 @implementation ProfileViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+    
+    self.recipesCollectionView.dataSource = self;
+    self.recipesCollectionView.delegate = self;
```

Comment:
this could also be done through storyboard. 
remind me during our 1:1 and i can show you how :)